### PR TITLE
Simplify env var lookup

### PR DESCRIPTION
**Pull request by vivid-synth**
This way we don't need to throw an exception

### CODE REVIEW

File: Sound/Tidal/Tempo.hs [hunk 1]
```diff
@@ -9,15 +9,17 @@ import Control.Monad (forM_, forever, void)
 import Control.Concurrent (forkIO, threadDelay)
 import Control.Concurrent.MVar
 import Control.Monad.Trans (liftIO)
+import Data.Maybe
```
Comment (lennart) [inline Sound/Tidal/Tempo.hs]:
```haskell
import Data.Maybe (fromMaybe,maybe)
```
 ?

Comment (vivid-synth) [inline Sound/Tidal/Tempo.hs]: K!

**Comment by lennart** [comment]
relates to #157

**Comment by lennart** [comment]
@vivid-synth if you update to latest 0.9-dev once again (I just merged the update to cabal 1.18) travis should pass!

**Comment by vivid-synth** [comment]
Updated

**Comment by lennart** [comment]
great, should be good to merge.

**Pull request merged**